docs/api/accordion.md

```diff
@@ -41,11 +41,19 @@ TODO  Playground
 
 ### Individual Accordion
 
-TODO Playground
+Individual accordions can be disabled with the `disabled` property on `ion-accordion`.
+
+import DisableIndividual from '@site/static/usage/accordion/disable-individual/index.md';
+
+<DisableIndividual />
```

Comment:
Could we make this playground a little taller? There's a scrollbar popping up if you have the first accordion open and then click the third in `md` mode.